dev.Dockerfile

```diff
@@ -34,6 +34,10 @@ COPY --from=builder /app/src ./src
 
 
 ENV PORT 8080
+
+HEALTHCHECK --interval=5s --timeout=2s --retries=10 \
+  CMD curl -f http://localhost:8080/healthcheck || exit 1
+
 EXPOSE $PORT
 
 ENTRYPOINT ["tini", "--"]
```

Comment:
Although we dont have DT on now should this have the env var set so if we add it back in future it works as expected?

Reply:
If I add the dynatrace layer will that not result in f2f dev stats appearing in dynatrace?